spec/requests/api/v1/provider_spec.rb

```diff
@@ -72,7 +72,6 @@ def get_provider_codes_from_body(body)
                region_code: :london,
                accrediting_provider: "Y",
                scheme_member: "Y",
-               last_published_at: DateTime.now.utc,
```

Comment:
We don't know what the effect this will have on UCAS. I'm not sure we should remove it.